samples/system-test/quickstart.test.js

```diff
@@ -17,42 +17,46 @@
 
 const proxyquire = require(`proxyquire`).noPreserveCache();
 const sinon = require(`sinon`);
-const test = require(`ava`);
+const assert = require(`assert`);
 const tools = require(`@google-cloud/nodejs-repo-tools`);
 
-test.before(tools.stubConsole);
-test.after.always(tools.restoreConsole);
+describe('QuickStart', () => {
+  before(() => tools.stubConsole);
+  after(() => tools.restoreConsole);
 
-test.cb(`should query a table`, t => {
-  const databaseMock = {
-    run: _query => {
-      t.deepEqual(_query, {
-        sql: `SELECT 1`,
-      });
-      setTimeout(() => {
+  it(`should query a table`, async () => {
+    const databaseMock = {
+      run: async _query => {
+        assert.deepStrictEqual(_query, {
+          sql: `SELECT 1`,
+        });
+
+        await new Promise(r => setTimeout(r, 200));
         try {
-          t.deepEqual(console.log.getCall(0).args, [`test`]);
-          t.end();
-        } catch (err) {
-          t.end(err);
-        }
-      }, 200);
-      return Promise.resolve([['test']]);
-    },
-  };
-  const instanceMock = {
-    database: sinon.stub().returns(databaseMock),
-  };
-  const spannerMock = {
-    instance: sinon.stub().returns(instanceMock),
-  };
-
-  proxyquire(`../quickstart`, {
-    '@google-cloud/spanner': {
-      Spanner: sinon.stub().returns(spannerMock),
-    },
-  });
+          assert.deepStrictEqual(console.log.getCall(0).args, [`test`]);
+        } catch (err) {}
```

Comment:
This err shouldn't get swallowed.  This should throw, and cause the test to fail.  I believe you can drop the try/catch.

Reply:
Change done